audio.js

```diff
@@ -1,12 +1,225 @@
+//Display audio element id
 const display = document.getElementById("display");
 
-$(".playback").click(function (e) {
-    e.preventDefault();
+//Power ON and OFF
+let toggle = document.getElementById("power-switch-input");
+display.innerHTML = 'Power Off';
 
-    var clip = $(this).next('audio').get(0);
+$('.playback').attr('disabled', 'disabled'); //audio off
 
-    display.innerHTML = clip.id;
+// console.log(typeof toggle.value);
 
-    clip.currentTime = 0;
-    clip.play();
+toggle.addEventListener('input', () => {
+    if ((toggle.value == '0' && bank.value == '0') || (toggle.value == '0' && bank.value == '1')) {
```

Comment:
This references the `bank` variable even though it hasn't been assigned yet. Recommend assigning it before using it.

But I also recommend not referencing it here at all. The power toggle event shouldn't be concerned with the bank toggle.

Reply:
bank variable is assigned on line 154 and it is heard on line 13. it was written this way to avoid the bank toggle from turning the audio back on.

Reply:
OK, good to know that there was a reason. However, the bank toggle shouldn't be capable of turning audio on or off. That should be the sole responsibility of the power toggle.